Doc/library/constants.rst

```diff
@@ -94,6 +96,13 @@ should not be used in programs.
    (i.e. EOF) to exit", and when called, raise :exc:`SystemExit` with the
    specified exit code.
 
+.. data:: help
+   :noindex:
+
+   Object that when printed, prints the message "Type help() for interactive
+   help, or help(object) for help about object.", and when called, invokes the
+   built-in :func:`help` function.
```

Comment:
The part beginning with 'invokes' is at least misleading when read as implying that there is or need-be a separate help function.
```suggestion
   help, or help(object) for help about object.", and when called,
   acts as described :func:`elsewhere <help>`.
```